packages/server/package.json

```diff
@@ -99,6 +99,7 @@
     "aws-sdk": "^2.99.0",
     "base64url": "^3.0.1",
     "bcrypt": "^5.0.0",
+    "busboy": "^0.3.1",
```

Comment:
busboy is used internally in `graphql-upload` and several other packages. supports streams as a first-class citizen.